interfaces/cython/cantera/test/test_purefluid.py

```diff
@@ -84,7 +84,7 @@ def test_X_deprecated(self):
         with warnings.catch_warnings(record=True) as w:
             warnings.simplefilter("always")
             X = self.water.X
-            self.water.TPX = 300, 101325, 1
+            self.water.TX = 300, 1
```

Comment:
Interesting that this passed before.
```
Psat at this T: 3528.213802290444
```

Reply:
The check for consistent TPQ in #784 (now in C++) is a little bit more stringent than what was in Python before. Before, if Q was 0 or 1, you could set any combination of T and P. The new check requires that Q == 1 for a vapor state and Q == 0 for a liquid state.